sdk/src/test/kotlin/com/pulumi/kotlin/options/AliasTest.kt

```diff
@@ -1,13 +1,10 @@
-package org.virtuslab.pulumikotlin.codegen.sdk.opts
+package com.pulumi.kotlin.options
```

Comment:
(minor)

Wouldn't it be better to move these tests to the `sdk` module now? Would it require more work that just moving these files?

Reply:
They are in the `sdk` module :D

Reply:
ok 👍 
I was just checking through GitHub interface and I missed that.